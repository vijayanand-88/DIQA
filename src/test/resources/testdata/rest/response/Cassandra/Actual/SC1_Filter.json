{
  "count" : 5,
  "results" : [ {
    "id" : "Default.Table:::7449",
    "type" : "Table",
    "name" : "table3",
    "asg.parentTypeNamePath" : [ "Cluster/dechehdpqa01v.asg.com/Service/CASSANDRA/Database/keyspace1" ],
    "asg.tagNames" : [ {
      "id" : "Default.tag:::23120",
      "name" : "Cassandra",
      "rootTagName" : "Technology",
      "parentTagName" : "",
      "definition" : "",
      "status" : "APPROVED",
      "type" : "Technology Tag",
      "image" : {
        "icon" : "icon-technology",
        "color" : "#35906c"
      },
      "userAssignable" : false,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-09-29T13:52:58.090+0000",
      "asg_createdat" : "2020-09-29T13:52:58.090+0000",
      "isRoot" : false,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 488,
      "asg_modifiedby" : "Service",
      "target" : 7449
    }, {
      "id" : "Default.tag:::23153",
      "name" : "SC1_CassandraFilter",
      "rootTagName" : "SC1_CassandraFilter",
      "parentTagName" : "",
      "definition" : "",
      "status" : "APPROVED",
      "type" : "Structure Information Tag",
      "userAssignable" : true,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-09-29T13:52:58.090+0000",
      "asg_createdat" : "2020-09-29T13:52:58.090+0000",
      "isRoot" : true,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 7474,
      "asg_modifiedby" : "Service",
      "target" : 7449
    }, {
      "id" : "Default.tag:::23186",
      "name" : "CassandraCatalogerBA",
      "rootTagName" : "BusinessApplication",
      "parentTagName" : "",
      "definition" : "BusinessApplication: CassandraCatalogerBA",
      "status" : "APPROVED",
      "type" : "BusinessApplication Tag",
      "image" : {
        "icon" : "icon-business",
        "color" : "#004ea8"
      },
      "userAssignable" : false,
      "subTags" : [ ],
      "isRoot" : false,
      "rootid" : null,
      "catalog" : null,
      "originItemID" : "Default.BusinessApplication:::2513",
      "source" : 2514,
      "target" : 7449,
      "asg_modifiedat" : "2020-09-29T13:52:58.090+0000",
      "asg_createdat" : "2020-09-29T13:52:58.090+0000",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "asg_modifiedby" : "Service",
      "originItemCatalog" : "Default"
    } ],
    "catalog" : {
      "name" : "Default",
      "label" : "Default"
    }
  }, {
    "id" : "Default.Table:::7452",
    "type" : "Table",
    "name" : "table1",
    "asg.parentTypeNamePath" : [ "Cluster/dechehdpqa01v.asg.com/Service/CASSANDRA/Database/keyspace2" ],
    "asg.tagNames" : [ {
      "id" : "Default.tag:::23141",
      "name" : "Cassandra",
      "rootTagName" : "Technology",
      "parentTagName" : "",
      "definition" : "",
      "status" : "APPROVED",
      "type" : "Technology Tag",
      "image" : {
        "icon" : "icon-technology",
        "color" : "#35906c"
      },
      "userAssignable" : false,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-09-29T13:52:58.090+0000",
      "asg_createdat" : "2020-09-29T13:52:58.090+0000",
      "isRoot" : false,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 488,
      "asg_modifiedby" : "Service",
      "target" : 7452
    }, {
      "id" : "Default.tag:::23174",
      "name" : "SC1_CassandraFilter",
      "rootTagName" : "SC1_CassandraFilter",
      "parentTagName" : "",
      "definition" : "",
      "status" : "APPROVED",
      "type" : "Structure Information Tag",
      "userAssignable" : true,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-09-29T13:52:58.090+0000",
      "asg_createdat" : "2020-09-29T13:52:58.090+0000",
      "isRoot" : true,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 7474,
      "asg_modifiedby" : "Service",
      "target" : 7452
    }, {
      "id" : "Default.tag:::23207",
      "name" : "CassandraCatalogerBA",
      "rootTagName" : "BusinessApplication",
      "parentTagName" : "",
      "definition" : "BusinessApplication: CassandraCatalogerBA",
      "status" : "APPROVED",
      "type" : "BusinessApplication Tag",
      "image" : {
        "icon" : "icon-business",
        "color" : "#004ea8"
      },
      "userAssignable" : false,
      "subTags" : [ ],
      "isRoot" : false,
      "rootid" : null,
      "catalog" : null,
      "originItemID" : "Default.BusinessApplication:::2513",
      "source" : 2514,
      "target" : 7452,
      "asg_modifiedat" : "2020-09-29T13:52:58.090+0000",
      "asg_createdat" : "2020-09-29T13:52:58.090+0000",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "asg_modifiedby" : "Service",
      "originItemCatalog" : "Default"
    } ],
    "catalog" : {
      "name" : "Default",
      "label" : "Default"
    }
  }, {
    "id" : "Default.Table:::7453",
    "type" : "Table",
    "name" : "table1",
    "asg.parentTypeNamePath" : [ "Cluster/dechehdpqa01v.asg.com/Service/CASSANDRA/Database/keyspace3" ],
    "asg.tagNames" : [ {
      "id" : "Default.tag:::23144",
      "name" : "Cassandra",
      "rootTagName" : "Technology",
      "parentTagName" : "",
      "definition" : "",
      "status" : "APPROVED",
      "type" : "Technology Tag",
      "image" : {
        "icon" : "icon-technology",
        "color" : "#35906c"
      },
      "userAssignable" : false,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-09-29T13:52:58.090+0000",
      "asg_createdat" : "2020-09-29T13:52:58.090+0000",
      "isRoot" : false,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 488,
      "asg_modifiedby" : "Service",
      "target" : 7453
    }, {
      "id" : "Default.tag:::23177",
      "name" : "SC1_CassandraFilter",
      "rootTagName" : "SC1_CassandraFilter",
      "parentTagName" : "",
      "definition" : "",
      "status" : "APPROVED",
      "type" : "Structure Information Tag",
      "userAssignable" : true,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-09-29T13:52:58.090+0000",
      "asg_createdat" : "2020-09-29T13:52:58.090+0000",
      "isRoot" : true,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 7474,
      "asg_modifiedby" : "Service",
      "target" : 7453
    }, {
      "id" : "Default.tag:::23210",
      "name" : "CassandraCatalogerBA",
      "rootTagName" : "BusinessApplication",
      "parentTagName" : "",
      "definition" : "BusinessApplication: CassandraCatalogerBA",
      "status" : "APPROVED",
      "type" : "BusinessApplication Tag",
      "image" : {
        "icon" : "icon-business",
        "color" : "#004ea8"
      },
      "userAssignable" : false,
      "subTags" : [ ],
      "isRoot" : false,
      "rootid" : null,
      "catalog" : null,
      "originItemID" : "Default.BusinessApplication:::2513",
      "source" : 2514,
      "target" : 7453,
      "asg_modifiedat" : "2020-09-29T13:52:58.090+0000",
      "asg_createdat" : "2020-09-29T13:52:58.090+0000",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "asg_modifiedby" : "Service",
      "originItemCatalog" : "Default"
    } ],
    "catalog" : {
      "name" : "Default",
      "label" : "Default"
    }
  }, {
    "id" : "Default.Table:::7450",
    "type" : "Table",
    "name" : "table2",
    "asg.parentTypeNamePath" : [ "Cluster/dechehdpqa01v.asg.com/Service/CASSANDRA/Database/keyspace1" ],
    "asg.tagNames" : [ {
      "id" : "Default.tag:::23145",
      "name" : "Cassandra",
      "rootTagName" : "Technology",
      "parentTagName" : "",
      "definition" : "",
      "status" : "APPROVED",
      "type" : "Technology Tag",
      "image" : {
        "icon" : "icon-technology",
        "color" : "#35906c"
      },
      "userAssignable" : false,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-09-29T13:52:58.090+0000",
      "asg_createdat" : "2020-09-29T13:52:58.090+0000",
      "isRoot" : false,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 488,
      "asg_modifiedby" : "Service",
      "target" : 7450
    }, {
      "id" : "Default.tag:::23178",
      "name" : "SC1_CassandraFilter",
      "rootTagName" : "SC1_CassandraFilter",
      "parentTagName" : "",
      "definition" : "",
      "status" : "APPROVED",
      "type" : "Structure Information Tag",
      "userAssignable" : true,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-09-29T13:52:58.090+0000",
      "asg_createdat" : "2020-09-29T13:52:58.090+0000",
      "isRoot" : true,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 7474,
      "asg_modifiedby" : "Service",
      "target" : 7450
    }, {
      "id" : "Default.tag:::23211",
      "name" : "CassandraCatalogerBA",
      "rootTagName" : "BusinessApplication",
      "parentTagName" : "",
      "definition" : "BusinessApplication: CassandraCatalogerBA",
      "status" : "APPROVED",
      "type" : "BusinessApplication Tag",
      "image" : {
        "icon" : "icon-business",
        "color" : "#004ea8"
      },
      "userAssignable" : false,
      "subTags" : [ ],
      "isRoot" : false,
      "rootid" : null,
      "catalog" : null,
      "originItemID" : "Default.BusinessApplication:::2513",
      "source" : 2514,
      "target" : 7450,
      "asg_modifiedat" : "2020-09-29T13:52:58.090+0000",
      "asg_createdat" : "2020-09-29T13:52:58.090+0000",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "asg_modifiedby" : "Service",
      "originItemCatalog" : "Default"
    } ],
    "catalog" : {
      "name" : "Default",
      "label" : "Default"
    }
  }, {
    "id" : "Default.Table:::7451",
    "type" : "Table",
    "name" : "table1",
    "asg.parentTypeNamePath" : [ "Cluster/dechehdpqa01v.asg.com/Service/CASSANDRA/Database/keyspace1" ],
    "asg.tagNames" : [ {
      "id" : "Default.tag:::23146",
      "name" : "Cassandra",
      "rootTagName" : "Technology",
      "parentTagName" : "",
      "definition" : "",
      "status" : "APPROVED",
      "type" : "Technology Tag",
      "image" : {
        "icon" : "icon-technology",
        "color" : "#35906c"
      },
      "userAssignable" : false,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-09-29T13:52:58.090+0000",
      "asg_createdat" : "2020-09-29T13:52:58.090+0000",
      "isRoot" : false,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 488,
      "asg_modifiedby" : "Service",
      "target" : 7451
    }, {
      "id" : "Default.tag:::23179",
      "name" : "SC1_CassandraFilter",
      "rootTagName" : "SC1_CassandraFilter",
      "parentTagName" : "",
      "definition" : "",
      "status" : "APPROVED",
      "type" : "Structure Information Tag",
      "userAssignable" : true,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-09-29T13:52:58.090+0000",
      "asg_createdat" : "2020-09-29T13:52:58.090+0000",
      "isRoot" : true,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 7474,
      "asg_modifiedby" : "Service",
      "target" : 7451
    }, {
      "id" : "Default.tag:::23212",
      "name" : "CassandraCatalogerBA",
      "rootTagName" : "BusinessApplication",
      "parentTagName" : "",
      "definition" : "BusinessApplication: CassandraCatalogerBA",
      "status" : "APPROVED",
      "type" : "BusinessApplication Tag",
      "image" : {
        "icon" : "icon-business",
        "color" : "#004ea8"
      },
      "userAssignable" : false,
      "subTags" : [ ],
      "isRoot" : false,
      "rootid" : null,
      "catalog" : null,
      "originItemID" : "Default.BusinessApplication:::2513",
      "source" : 2514,
      "target" : 7451,
      "asg_modifiedat" : "2020-09-29T13:52:58.090+0000",
      "asg_createdat" : "2020-09-29T13:52:58.090+0000",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "asg_modifiedby" : "Service",
      "originItemCatalog" : "Default"
    } ],
    "catalog" : {
      "name" : "Default",
      "label" : "Default"
    }
  } ],
  "configs" : {
    "defaultListView" : [ {
      "value" : "asg.parentTypeNamePath",
      "label" : "Hierarchy"
    }, {
      "value" : "asg.tagNames",
      "label" : "Tags"
    } ]
  },
  "configsMap" : {
    "*" : {
      "*" : "defaultListView"
    }
  },
  "facets" : {
    "asg.tagPathsHierarchy_ss" : {
      "label" : "Tags",
      "type" : "TREE",
      "values" : [ {
        "count" : 5,
        "value" : "BusinessApplication",
        "label" : "BusinessApplication",
        "checked" : false,
        "childValues" : [ {
          "count" : 5,
          "value" : "BusinessApplication/CassandraCatalogerBA",
          "label" : "CassandraCatalogerBA",
          "checked" : false,
          "childValues" : [ ]
        } ]
      }, {
        "count" : 5,
        "value" : "SC1_CassandraFilter",
        "label" : "SC1_CassandraFilter",
        "checked" : true,
        "childValues" : [ ]
      }, {
        "count" : 5,
        "value" : "Technology",
        "label" : "Technology",
        "checked" : false,
        "childValues" : [ {
          "count" : 5,
          "value" : "Technology/Enterprise Data",
          "label" : "Enterprise Data",
          "checked" : false,
          "childValues" : [ {
            "count" : 5,
            "value" : "Technology/Enterprise Data/NoSQL Databases",
            "label" : "NoSQL Databases",
            "checked" : false,
            "childValues" : [ {
              "count" : 5,
              "value" : "Technology/Enterprise Data/NoSQL Databases/Cassandra",
              "label" : "Cassandra",
              "checked" : false,
              "childValues" : [ ]
            } ]
          } ]
        } ]
      } ]
    },
    "type_s" : {
      "label" : "Metadata Type",
      "type" : "FLAT_LIST",
      "values" : [ {
        "count" : 15,
        "value" : "Column",
        "checked" : false
      }, {
        "count" : 5,
        "value" : "Constraint",
        "checked" : false
      }, {
        "count" : 5,
        "value" : "Table",
        "checked" : true
      }, {
        "count" : 3,
        "value" : "Database",
        "checked" : false
      }, {
        "count" : 1,
        "value" : "Service",
        "checked" : false
      } ]
    },
    "asg.parentTypeNamePathHierarchy_ss" : {
      "label" : "Hierarchy",
      "type" : "TREE",
      "values" : [ {
        "count" : 5,
        "value" : "Cluster/dechehdpqa01v.asg.com",
        "label" : "dechehdpqa01v.asg.com [Cluster]",
        "checked" : false,
        "childValues" : [ {
          "count" : 5,
          "value" : "Cluster/dechehdpqa01v.asg.com/Service/CASSANDRA",
          "label" : "CASSANDRA [Service]",
          "checked" : true,
          "childValues" : [ {
            "count" : 3,
            "value" : "Cluster/dechehdpqa01v.asg.com/Service/CASSANDRA/Database/keyspace1",
            "label" : "keyspace1 [Database]",
            "checked" : true,
            "childValues" : [ ]
          }, {
            "count" : 1,
            "value" : "Cluster/dechehdpqa01v.asg.com/Service/CASSANDRA/Database/keyspace2",
            "label" : "keyspace2 [Database]",
            "checked" : true,
            "childValues" : [ ]
          }, {
            "count" : 1,
            "value" : "Cluster/dechehdpqa01v.asg.com/Service/CASSANDRA/Database/keyspace3",
            "label" : "keyspace3 [Database]",
            "checked" : true,
            "childValues" : [ ]
          } ]
        } ]
      } ]
    },
    "asg_modifiedat_dt" : {
      "label" : "Last Modified",
      "type" : "FLAT_LIST",
      "values" : [ {
        "count" : 0,
        "value" : "(*,NOW-1YEAR]",
        "label" : "1 year and older",
        "checked" : false
      }, {
        "count" : 0,
        "value" : "(NOW-1YEAR,NOW-6MONTH]",
        "label" : "6 months up to 1 year",
        "checked" : false
      }, {
        "count" : 0,
        "value" : "(NOW-6MONTH,NOW-1MONTH]",
        "label" : "1 month up to 6 months",
        "checked" : false
      }, {
        "count" : 0,
        "value" : "(NOW-1MONTH,NOW-7DAY]",
        "label" : "1 week up to 1 month",
        "checked" : false
      }, {
        "count" : 0,
        "value" : "(NOW-7DAY,NOW-1DAY]",
        "label" : "1 day up to 1 week",
        "checked" : false
      }, {
        "count" : 5,
        "value" : "(NOW-1DAY,NOW]",
        "label" : "past 24 hours",
        "checked" : false
      } ]
    },
    "asg_modifiedby_s" : {
      "label" : "Modified by",
      "type" : "FLAT_LIST",
      "values" : [ {
        "count" : 5,
        "value" : "Service",
        "checked" : false
      } ]
    }
  }
}