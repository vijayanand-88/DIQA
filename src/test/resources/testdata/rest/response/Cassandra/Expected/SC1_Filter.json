{
  "SC1": {
    "Tables": [
      "table2",
      "table1",
      "table1",
      "table3",
      "table1"
    ]
  },
  "SC2": {
    "Tables": [
      "table2",
      "table1",
      "table1",
      "table5",
      "table1"
    ]
  },
  "SC3": {
    "Tables": [
      "table1",
      "table5",
      "table1",
      "table4"
    ]
  },
  "SC4": {
    "Tables": [
      "table5",
      "table1",
      "table1",
      "table2",
      "table1"
    ]
  }
}